recipes/jenkspy/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "jenkspy" %}
+{% set version = "0.1.3" %}
```

Comment:
It looks like v `0.1.4` is available. Unless you need this earlier version, can you bump the version?